core/src/main/java/io/aiven/klaw/config/ConfigUtils.java

```diff
@@ -31,6 +31,8 @@ protected static List<String> getStaticResources(boolean coralEnabled) {
                 "/getDbAuth",
                 "/feedback**",
                 "/resetPassword",
```

Comment:
can we delete this ?

Reply:
Yes I should have removed this endpoint as it is now redundant.
Will get that and all its pieces now.